ios/Gekidou/Sources/Gekidou/Networking/Network.swift

```
@@ -33,6 +37,39 @@ public class Network: NSObject {
         )
     }
     
+    internal func loadPinnedCertificates() {
+        if let certsPath = Bundle.app.resourceURL?.appendingPathComponent("certs") {
```

Minimize indentation, fail fast, and improve readability by:

```
guard let certsPath = Bundle.app.resourceURL?.appendingPathComponent("certs") else {
 	return
 }

// happy path
```